docs/Project.toml

```diff
@@ -1,5 +1,17 @@
 [deps]
+Distributions = "31c24e10-a181-5473-b8eb-7969acd0382f"
 Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
+Mimi = "e4e893b0-ee5e-52ea-8111-44b3bdec128c"
+MimiDICE2010 = "4f2c5d70-42a7-11e9-1de8-c370bd984083"
+MimiFUND = "b3ba11de-429f-11e9-29f7-cb478ab96e7c"
+StatsBase = "2913bbd2-ae8a-5f71-8c99-4fb6c76f3a91"
+VegaLite = "112f6efa-9a02-5b7d-90c0-432ed331239a"
 
 [compat]
-Documenter = "~0.21"
+Distributions = "0.21"
+Documenter = "0.23"
+Mimi = "0.9.3"
```

Comment:
should this be Mimi 0.9.4?

Reply:
Or actually wouldn't we need it to be the 0.9.5-dev version? I guess this is a broader question I have about whether or not our documentation is supposed to be compatible with the latest released version, or with master. I _think_ that the version of documentation that lives on https://www.mimiframework.org/Mimi.jl/stable/ only shows the documentation for the latest _released_ version right? So that means we should always have our documentation up to date with master, but it won't actually display until a new release of Mimi happens? 

If all of that is true, then I think the version of Mimi that we want in the Project.toml would be master, but I'm not sure how to specify that

Reply:
Yes, the version of the documentation that lives in the `stable` link only shows documentation for the latest **tagged/released** version of Mimi, so the documentation might as well keep up with master.

Reply:
I think you should remove all entries other than `Documenter` itself from here. Documenter should be specified as `~0.23`.

Reply:
What does the `~` mean? I think we also need to keep MimiFUND at at least `3.11.9`, so would that also need to be `MimiFUND = "~3.11.9"` here?

Reply:
https://julialang.github.io/Pkg.jl/v1/compatibility/#Caret-specifiers-1

So I actually think it doesn't matter whether we use caret or tilde for `Documenter`, we should just specify one of them.

For FUND, if there is a minimal version we require we can just say `MimiFUND = "3.11.9"`. But do we need that? It will just grab the latest version if we don't specify anything, which is what we want, right?

Reply:
Ok, but for FUND, I think we should actually _not_ use the tilde operator, otherwise we potentially pin FUND to old versions.

Reply:
If it will always grab the most recent version, then why do we need to specify for Documenter?

Reply:
Ok, we shouldn't have to specify FUND then. But why do we have to specify for Documenter then?

Reply:
They might release a breaking release, and we would want to opt in to get that.